src/components/FeatureMarkerTable/components/MarkerTableGeneCell.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import * as R from 'ramda';
+import { withRouter, RouteComponentProps } from 'react-router-dom';
+
+import { BackendAPI } from '../../common/API';
+
+import { makeTableColumnData } from '../model';
+
+type GeneTableCellProps = {
+    history: any;
+    activePage: string;
+    activeFeature: any;
+    activeFeatureIndex: number;
+    value: string;
+} & RouteComponentProps<{ uuid: string; loom: string }>;
+
+class GeneTableCell extends React.Component<GeneTableCellProps> {
```

Comment:
Can this be a function component?

Reply:
Damn, I just noticed that there is a statement that is setting the state. I probably skipped this line when I was refactoring at that time..

Reply:
Actually it can be removed since it's this property is even not used in `ViewerSideBar.jsx` on `master`